app/models/activity.rb

```diff
@@ -12,6 +12,11 @@ def time_formatted
     created_at.strftime("%-I:%M %p")
   end
 
+  def whodunnit
```

Comment:
Do we need this `whodunnit` bit? We should be accessing the user record by the `User` association on `Activity`. In the SQL view, we map `whodunnit` to `user_id` to make this work.